compiler/z/codegen/S390Peephole.cpp

```diff
@@ -2046,7 +2046,7 @@ TR_S390PostRAPeephole::revertTo32BitShift()
    // Note the NOT in front of second boolean expr. pair
    TR::InstOpCode::Mnemonic oldOpCode = instr->getOpCodeValue();
    if ((oldOpCode == TR::InstOpCode::SLLG || oldOpCode == TR::InstOpCode::SLAG)
-         && !(instr->getNode()->getOpCodeValue() == TR::ishl || instr->getNode()->getOpCodeValue() == TR::iushl))
+         && !instr->getNode()->getOpCodeValue() == TR::ishl)
```

Comment:
This looks like a bug. I think you meant:

```
&& instr->getNode()->getOpCodeValue() != TR::ishl
```

I suspect this is the reason we have PR build failures on Linux on Z.

Reply:
Hooray for Tril unit testing by the way. Not long ago this would have slid by unnoticed!

Reply:
Right, I should have done `&& !(instr->getNode()->getOpCodeValue() == TR::ishl)` or just `&& instr->getNode()->getOpCodeValue() != TR::ishl`. Fixed that in the new force push.